tests/integration/linux_bridge_test.py

```diff
@@ -397,6 +397,111 @@ def test_port_vlan_not_implemented(port0_up):
         pass
 
 
+@pytest.mark.xfail(
+    reason='https://nmstate.atlassian.net/browse/NMSTATE-230', strict=True
+)
+class TestVlanFiltering(object):
+    access_tag = 300
+    bridge_name = TEST_BRIDGE0
```

Comment:
To access these class variables, the proper way is to use <class-name>.<variable-name>.
It will be easier to just promote these constants to the module level (as constants).

Reply:
I don't understand the first sentence (... proper way is to use ..). What does '..' do ? 

I sincerely believe these belong in the class, since they're related to a specific feature of linux bridges (all other tests not in this class couldn't care less about them).

Reply:
I meant that static class variables should be accessed through the class name and not `self` because the latter is confusing the reader, thinking it is an instance variable.
I do not mind if you leave them here or promote them to the module level, that is up to you.